### PR TITLE
Fix broken tests in PHP 5.5

### PR DESCRIPTION
**Pull request by rclations**
(still in progress)

### CODE REVIEW

**Comment by rclations** [comment]
for #1209

**Comment by rclations** [comment]
all checks passing! read to go!

File: tests/inc/test-update.php [hunk 1]
```diff
@@ -318,6 +318,11 @@ function test_largo_transition_nav_menus() {
 	}
 
 	function test_largo_update_prominence_term_description_single() {
+		$term9 = get_term_by('slug', 'term-9', 'prominence', 'ARRAY_A');
+		if ( ! $term9 ) {
+			wp_insert_term( 'term-9', 'prominence', array( 'description' => 'Term Description 9' ) );
```
Comment (benlk) [inline tests/inc/test-update.php]: We shouldn't have to create this term; it should have been created by this class' `setUp` method: https://github.com/rclations/Largo/blob/a773c5a2cfdbb7706c41085e8388f0d83b583e8e/tests/inc/test-update.php#L29-L31

Comment (benlk) [inline tests/inc/test-update.php]: Previous comment might be wrong, based on https://github.com/INN/Largo/issues/1209#issuecomment-225950908

Either way, the test tests what it's supposed to test.

**Pull request merged**